docs/hw/jetson_hookup.md

```diff
@@ -0,0 +1,29 @@
+# iRobot® Create® 3 Hookup Guide for NVIDIA® Jetson™
+
+The iRobot® Create® 3 has a USB-C®[^1] connector implementing a USB 2.0 host capable of 5 V @ 3 A continuous, which can be used to power and communicate with various downstream devices.
+Here are some hookup examples for NVIDIA® Jetson™[^2] computers.
+
+## NVIDIA® Jetson Nano™ 2GB
+The Jetson Nano™ 2GB has a USB-C® port (J2) for power and a USB Micro-B port (J13) for downstream data.
+This can be connected to the Create 3 most simply using a USB-C® hub and two cables -- USB A to Micro-B and USB A to USB-C®.
```

Comment:
"the Create 3 robot most simply" (should have been caught before)

Reply:
Fixed.